sim/mcuboot-sys/build.rs

```diff
@@ -81,6 +82,64 @@ fn main() {
         panic!("mcuboot does not support more than one sig type at the same time");
     }
 
+    if psa_crypto_api {
+        if sig_ecdsa || enc_ec256 || enc_x25519 || enc_aes256_ec256 || sig_ecdsa_mbedtls || enc_aes256_x25519 || enc_kw  || enc_aes256_kw {
```

Comment:
Please break this into multiple lines, most of the code does not exceed 80 characters.